code/modules/recycling/sortingmachinery.dm

```diff
@@ -163,11 +163,11 @@
 	resistance_flags = FLAMMABLE
 
 /obj/item/stack/packageWrap/afterattack(var/obj/target as obj, mob/user as mob, proximity)
+	var/static/list/no_wrap = list(/obj/item/smallDelivery, /obj/structure/bigDelivery, /obj/item/evidencebag, /obj/structure/closet/body_bag, /obj/item/twohanded/required)
```

Comment:
I feel like this list is better on the object itself, instead of being defined in the proc. Yes I know its a static, but it makes more sense to define it at the object level.